src/header/header.scss

```diff
@@ -3,441 +3,61 @@
 @import '../style/index';
 @import '../user-icon/index';
 @import '../icon/index';
-@import '../button/index';
 
 @mixin iui-page-header {
   @include iui-reset;
-
   display: flex;
   justify-content: space-between;
   width: 100%;
   height: $iui-baseline * 5 + $iui-xxs; // Remove `+ $iui-xxs` when box-sizing: border-box is applied to all elements.
   user-select: none;
   box-sizing: border-box;
-  // iPhone notch support
-  padding: env(safe-area-inset-top) env(safe-area-inset-right) 0 env(safe-area-inset-left);
+  background-color: var(--iui-color-background-1);
+  border-bottom: $iui-xxs solid var(--iui-color-background-border);
+  color: var(--iui-text-color);
+  padding: env(safe-area-inset-top) env(safe-area-inset-right) 0 env(safe-area-inset-left); // iPhone notch support
+}
+
+@mixin iui-page-header-slim {
   @media (prefers-reduced-motion: no-preference) {
```

Comment:
My thinking here was that the `transition` isn't needed unless you have the option to make it slim.  And the transition needs to be applied on `.iui-page-header`, can't be applied on `.iui-slim` or it would only animate when going from slim to thick.

Reply:
and how we choose if we have option to make it slim or not?
Not this mixin is by default included under `iui-page-header`.
So this transition rule is always there.